src/components/SearchModal/CurrencySearch.tsx

```diff
@@ -126,7 +126,7 @@ export function CurrencySearch({
     const s = debouncedQuery.toLowerCase().trim()
     if (native.symbol?.toLowerCase()?.indexOf(s) !== -1) {
       // Always bump the native token to the top of the list.
-      return native ? [native, ...filteredSortedTokens.filter((t) => !t.equals(native))] : filteredSortedTokens
+      return [native, ...filteredSortedTokens.filter((t) => !t.equals(native))]
```

Comment:
would this potentially return undefined at the front if native is not defined?

Reply:
if native is undefined this code wouldn't be reached, would return out of the fn on line 123